VultisigApp/VultisigApp/Model/TransactionMemo/TransactionMemoBond.swift

```diff
@@ -56,7 +56,7 @@ class TransactionMemoBond: TransactionMemoAddressable, ObservableObject {
     
     private func setupValidation() {
         Publishers.CombineLatest4($amountValid, $nodeAddressValid, $providerValid, $feeValid)
-            .map { $0 && $1 && $2 && $3 }
+            .map { $0 && $1 && $2 && $3 && (!self.provider.isEmpty ? self.fee != .zero : true) }
```

Comment:
you can make isTheFormValid computed property: 
```
var isTheFormValid: Bool {
return amountValid && nodeAddressValid ...
}
```

Reply:
Since we are using publishers, it's not possible to directly convert this logic into a computed property because computed properties do not support reactive updates or subscriptions to publishers. Instead, we typically use a @Published property to store the result of the combined validation logic and update it reactively.